### PR TITLE
update version to 3.4.3

### PR DESCRIPTION
**Pull request by pdabelf5**
### Proposed changes

update version to 3.4.3

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that all unit tests pass after adding my changes
- [x] I have updated necessary documentation
- [x] I have rebased my branch onto main
- [ ] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

### CODE REVIEW

**Review by jjngx — APPROVED** [review]

**Review by vepatel — APPROVED** [review]

**Review by oseoin — APPROVED** [review]

File: docs/content/installation/nic-images/using-the-jwt-token-docker-secret.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ If the namespace does not exist, `--create-namespace` will create it. Using `-f
 If you want to install NGINX Ingress Controller using the charts method, the following is an example of using the command line to pass the required arguments using the `set` parameter.
 
 ```shell
-helm install my-release -n nginx-ingress oci://ghcr.io/nginxinc/charts/nginx-ingress --version 1.1.2 --set controller.image.repository=private-registry.nginx.com/nginx-ic/nginx-plus-ingress --set controller.image.tag=3.4.2 --set controller.nginxplus=true --set controller.serviceAccount.imagePullSecretName=regcred
+helm install my-release -n nginx-ingress oci://ghcr.io/nginxinc/charts/nginx-ingress --version 1.1.2 --set controller.image.repository=private-registry.nginx.com/nginx-ic/nginx-plus-ingress --set controller.image.tag=3.4.3 --set controller.nginxplus=true --set controller.serviceAccount.imagePullSecretName=regcred
```
Comment (sigv) [inline docs/content/installation/nic-images/using-the-jwt-token-docker-secret.md:153]: This line still references Helm chart `--version 1.1.2` although release 3.4.3 is expected to be used via chart version 1.1.3.

Comment (sigv) [reply docs/content/installation/nic-images/using-the-jwt-token-docker-secret.md:153]: Similarly, [`charts/nginx-ingress/README.md` L90](https://github.com/nginxinc/kubernetes-ingress/pull/5128/files#diff-0ba161dd052247c6e21fb8101c5979dfa0eef87ef0632a16f572560e76ccd087L90-R90) can be observed as missing a replacement. However, [`docs/content/technical-specifications.md`](https://github.com/nginxinc/kubernetes-ingress/pull/5128/files#diff-3e4bd30977f036fa4d02dea9cbd959d65f02aff5d2792e855f9b530f410d91c7L26-R26) was updated (by hand?).

Comment (sigv) [reply docs/content/installation/nic-images/using-the-jwt-token-docker-secret.md:153]: What's strange to me with this is that `helm pull oci://ghcr.io/nginxinc/charts/nginx-ingress --untar --version 1.1.3` produces an `nginx-ingress/README.md` where `1.1.3` is mentioned.

Comment (danielnginx) [reply docs/content/installation/nic-images/using-the-jwt-token-docker-secret.md:153]: @sigv great catch we appreciate it. We are going to fix it.

Comment (danielnginx) [reply docs/content/installation/nic-images/using-the-jwt-token-docker-secret.md:153]: https://github.com/nginxinc/kubernetes-ingress/pull/5287

**Pull request merged**